src/library_sdl.js

```diff
@@ -2983,6 +2983,11 @@ var LibrarySDL = {
   TTF_OpenFont: function(filename, size) {
     filename = PATH.normalize(Pointer_stringify(filename));
     var id = SDL.fonts.length;
+    if (filename != 'serif' && filename != 'sans-serif' &&
+        filename != 'cursive' && filename != 'fantasy' &&
+        filename != 'monospace') {
+      Runtime.warnOnce("TTF_OpenFont may not work correctly with custom font, please use one of ['serif', 'sans-serif', 'cursive', 'fantasy', 'monospace']");
```

Comment:
This is generic font-family names (https://www.w3.org/TR/CSS2/fonts.html#generic-font-families). Maybe we need add to filename some generic font family name to fallback. Like this:

```'"' + filename +'", serif' ```

What do you think?